woocommerce/woocommerce-ios
# Order Fulfillment: give order item some padding when no SKU

Closes #1359 

Add more space below image when no SKU exists. (Tested using an iPhone 11 Pro Max since it is more noticeable when autolayout changes are set incorrectly.)

| Before | After |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/1062444/70477426-7b0fa800-1a9e-11ea-842b-19f5fda59c57.png" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/70541690-1e0f0300-1b2d-11ea-9282-b7c24bff684f.png" width="350" /> |

### To test
1. Navigate to a processing order, where the order items do not have the SKU field filled out. Select the "Begin fulfillment" button to navigate to the Fulfillment screen.
2. Check the cell separator underneath the image.

Update release notes:
